### ISSUE TITLE
[Bug]: inpaint+llama producing dark/black results 

### ISSUE THREAD

**Issue by Anan-Saadi** [Mikubill/sd-webui-controlnet#2143]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

recently when I try to use inpaint+llama control net the output is very dark and sometimes black, and this only occurs when with inapint+llama, other modes work properly, and the seems to be exaggerated when paired with reference controlnet:

input image:
![copertina-1](https://github.com/Mikubill/sd-webui-controlnet/assets/88707312/519fe585-0367-4b10-874a-389e7840e698)

I masked the lower area of the image with prompt "cliff, rocks, stone", and general negative prompts like "ugly, blur..." 
all the below images were generated with "masked mode" set to "latent nothing", but I did confirm that the issue persists with all modes, except latent noise which seems to be a bit better but still not perfect 
no controlnet :
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/88707312/1db77e41-90c8-41c8-8977-e48ba3f9da4f)

inpaint + llama:
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/88707312/fce2c0dd-f687-48aa-a159-c020a2eab245)

inpaint+llama + refrence_only:
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/88707312/c5a37a6f-ac21-4c98-9a3e-f68eb7f8d000)

inpaint_only:
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/88707312/883b5f02-834a-48aa-811d-0790b826fd6d)

### Steps to reproduce the problem

1. launch webui and select any image 
2. draw any mask 
3. select controlnet inapint+llama
4. generate image 

### What should have happened?

a better non dark image should've been produced 

### Commit where the problem happens

webui: 525687fb69 (v1.6.0)
controlnet: 7a4805c8 (latest at the time of writing)

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
--xformers --skip-torch-cuda-test --api
```

### List of enabled extensions

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/88707312/2bc6b143-ba8a-416f-b423-7bb4d2968ad9)

### Console logs

```Shell
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: v1.6.0
Commit hash: 5ef669de080814067961f28357256e8fe27544f4
Fetching updates for midas...
Checking out commit for midas with hash: 1645b7e...
Launching Web UI with arguments: --xformers --skip-torch-cuda-test --api
2023-09-24 15:57:07,682 - ControlNet - INFO - ControlNet v1.1.410
ControlNet preprocessor location: E:\VS projects\stable diffusion\stable-diffusion-webui\extensions\sd-webui-controlnet\annotator\downloads
2023-09-24 15:57:09,389 - ControlNet - INFO - ControlNet v1.1.410
Loading weights [0f1b80cfe8] from E:\VS projects\stable diffusion\stable-diffusion-webui\models\Stable-diffusion\dreamshaperXL10_alpha2Xl10.safetensors
Creating model from config: E:\VS projects\stable diffusion\stable-diffusion-webui\repositories\generative-models\configs\inference\sd_xl_base.yaml
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 183.0s (prepare environment: 14.1s, launcher: 0.2s, import torch: 48.8s, import gradio: 15.2s, setup paths: 12.1s, import ldm: 0.1s, initialize shared: 1.7s, other imports: 9.9s, setup SD model: 0.1s, setup codeformer: 1.8s, setup gfpgan: 0.4s, list SD models: 3.7s, load scripts: 58.3s, load upscalers: 0.1s, reload hypernetworks: 0.2s, initialize extra networks: 0.9s, create ui: 13.8s, gradio launch: 1.3s, add APIs: 0.1s, app_started_callback: 1.6s).
Applying attention optimization: xformers... done.
Model loaded in 110.3s

2023-09-24 19:10:01,694 - ControlNet - INFO - Loading model from cache: control_v11p_sd15_inpaint [ebff9138]
2023-09-24 19:10:01,939 - ControlNet - INFO - Loading preprocessor: inpaint_only+lama
2023-09-24 19:10:01,940 - ControlNet - INFO - preprocessor resolution = 338
2023-09-24 19:10:03,248 - ControlNet - INFO - ControlNet used torch.float16 VAE to encode torch.Size([1, 4, 42, 75]).
2023-09-24 19:10:03,282 - ControlNet - INFO - ControlNet Hooked - Time = 1.588486671447754
2023-09-24 19:10:03,694 - ControlNet - INFO - [ControlNet] Initial noise hack applied to torch.Size([1, 4, 42, 75]).
  0%|                                                                                           | 0/31 [00:00<?, ?it/s]2023-09-24 19:10:03,860 - ControlNet - INFO - ControlNet used torch.float16 VAE to encode torch.Size([1, 4, 42, 75]).
 
 *generation completes with  no errors* 

```

### Additional information

I am using inapint model control_v11p_sd15_inpaint [ebff9138]
things I attempted to fix the issue:

- I tried multiple models, inpaint and non inpaint 
- I tried deleting and reinstalling the venv folder 
- I tried generating the image via api and still received an identical image

**Comment by huchenlei** [comment]
I think the result is expected as lama inpaint preprocessor probably filled the masked area with black first, then no matter what you put in your prompt, the result is based on that black preprocessed image.
